aws/components/sqs/state.ftl

```diff
@@ -36,6 +36,28 @@
         [#local dlqId = formatDependentResourceId(AWS_SQS_RESOURCE_TYPE, id, "dlq") ]
         [#local dlqName = formatName(name, "dlq")]
 
+        [#-- override the Id ane Name for replacement --]
+        [#if ((commandLineOptions.Deployment.Unit.Alternative)!"") == "replace1" ]
```

Comment:
This will break https://github.com/hamlet-io/engine-plugin-aws/pull/255 - needs to use `getDeploymentUnitAlternative()` instead